features/fusetools.adoc

```diff
@@ -7,7 +7,7 @@
 :page-feature_tagline: Helps get you over the hump
 :page-issues_url: https://issues.jboss.org/projects/FUSETOOLS
 
-== Fuse Tooling
+== [Deprecated] Fuse Tooling
```

Comment:
also to add the red ribon on the feature page, you must add
`:page-feature_deprecated: true`
above